mpt/impl/imap-mailbox/core/src/main/java/org/apache/james/mpt/host/JamesImapHostSystem.java

```diff
@@ -57,7 +57,7 @@
 
     @Override
     public void beforeTest() throws Exception {
-        memoryUsersRepository = MemoryUsersRepository.withoutVirtualHosting();
+        memoryUsersRepository = MemoryUsersRepository.withoutVirtualHosting(null);
```

Comment:
Maybe should be consistent with the other test files, by declaring a constant `NO_DOMAIN_LIST` for null? (or maybe a fixture?)